Library/Homebrew/formula.rb

```diff
@@ -223,6 +223,28 @@ def resource(name)
     active_spec.resource(name)
   end
 
+  # An old name for the formula
+  def oldname
+    return @oldname if @oldname
+    if core_formula?
+      begin
+        load "formula_renames.rb"
+      rescue LoadError
+      end
```

Comment:
Please remove this and put it at the top of `formula.rb` as `require "formula_renames"`.